src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String letterA;
+    private int variableA;
```

Comment:
What does `variable` mean?

Reply:
absolutely agree, not the best name for any variable, but from the point of view of the regular expression, it is correct.
https://regex101.com/r/sJ9gM7/796